### ISSUE TITLE
split report empty

### ISSUE THREAD

**Issue by christianmerges** [openaudible/openaudible#1410]
The split report looks like this after canceling the process:

```
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **
    ** Split Results **


Good:112, Errors:3 Skipped:4439
Destination: /config/OpenAudible/books
```

After update from 4.2 to 4.2.1

**Comment by openaudible** [comment]
Can you look at the File: Log Window? 
That’s a lot of books to attempt to split. Maybe find one or two to split… usually the book doesn’t have chapters…  but the report is definitely not right. You can send the log to us to info at OpenAudible dot org.

**Comment by openaudible** [comment]
Good catch. Fixed in beta https://openaudible.org/beta and will be fixed in 4.2.2.

**Comment by ZLoth** [comment]
Thank you!
